flask_rebar/swagger_generation/swagger_generator_v2.py

```diff
@@ -287,7 +287,12 @@ def _get_paths(self, paths, default_headers_schema, default_security=None):
                 }
 
                 if d.func.__doc__:
-                    path_definition[method_lower][sw.description] = d.func.__doc__
+                    docstring = d.func.__doc__.strip()
+                    doc_parts = str.split(docstring, "\n\n", 1)
+                    path_definition[method_lower][sw.summary] = doc_parts[0]
```

Comment:
This already breaks backward compatibility, but still we may want to assign the docstring to the description when there is only 1 part instead of the summary so that it's a slightly less breaking change.